packages/babel-template/src/populate.js

```diff
@@ -15,7 +15,14 @@ export default function populatePlaceholders(
       if (
         !Object.prototype.hasOwnProperty.call(replacements, placeholder.name)
       ) {
-        throw new Error(`No substitution given for "${placeholder.name}"`);
+        const placeholderName = placeholder.name;
+
+        throw new Error(
+          `Error: No substitution given for "${placeholderName}". If this is not meant to be a
```

Comment:
multiline string like this will look weirdly in the console, personally I would prefer using regular concatenation for this

Reply:
fair point.  was too focused on the struggle to step back and consider the alternatives.